src/main/webapp/app/service/trial.service.ts

```diff
@@ -147,7 +160,7 @@ export class TrialService {
                 wildtype: '',
                 no_hugo_symbol: false,
                 no_annotated_variant: false,
-                no_matching_examples: false,
```

Comment:
We don't need this anymore?

Reply:
No. matching_example doesn't have Not checkbox.